doc/userguide/rules/http-keywords.rst

```diff
@@ -1,813 +1,885 @@
-http.request_line              http_request_line (*)    Request
-http.request_body              http_client_body         Request
-http.header                    http_header              Both
-http.header.raw                http_raw_header          Both
-http.cookie                    http_cookie              Both
-http.user_agent                http_user_agent          Request
-http.host                      http_host                Request
-http.host.raw                  http_raw_host            Request
-http.accept                    http_accept (*)          Request
-http.accept_lang               http_accept_lang (*)     Request
-http.accept_enc                http_accept_enc (*)      Request
-http.referer                   http_referer (*)         Request
-http.connection                http_connection (*)      Both
-file.data                      file_data (*)            Both
-file.name                      filename (*)             Request
-http.content_type              http_content_type (*)    Both
-http.content_len               http_content_len (*)     Both
-http.start                     http_start (*)           Both
-http.protocol                  http_protocol (*)        Both
-http.header_names              http_header_names (*)    Both
-============================== ======================== ==================
-
-\*) sticky buffer
-
-The following **response** keywords are available:
-
-============================== ======================== ==================
-Keyword                        Legacy Content Modifier  Direction
-============================== ======================== ==================
-http.stat_msg                  http_stat_msg            Response
-http.stat_code                 http_stat_code           Response
-http.response_line             http_response_line (*)   Response
-http.header                    http_header              Both
-http.header.raw                http_raw_header          Both
-http.cookie                    http_cookie              Both
-http.response_body             http_server_body         Response
-http.server                    N/A                      Response
-http.location                  N/A                      Response
-file.data                      file_data (*)            Both
-http.content_type              http_content_type (*)    Both
-http.content_len               http_content_len (*)     Both
-http.start                     http_start (*)           Both
-http.protocol                  http_protocol (*)        Both
-http.header_names              http_header_names (*)    Both
-============================== ======================== ==================
-
-\*) sticky buffer
+Using the HTTP specific sticky buffers (see :ref:`rules-modifiers`) provides a
+way to efficiently inspect the specific fields of HTTP protocol communications.
+After specifying a sticky buffer in a rule it should be followed by one or
+more :doc:`payload-keywords`.
 
```

Comment:
I've noticed that the notes about corresponding PCRE modifiers were removed. Was wondering if adding just one mention to that, maybe in this section, pointing to Suricata's modifiers (https://docs.suricata.io/en/latest/rules/payload-keywords.html#suricata-s-modifiers)  would be a good idea?